crypto/src/lib.rs

```diff
@@ -70,13 +76,15 @@ use openmls_traits::OpenMlsCryptoProvider;
 use std::collections::HashMap;
 use tls_codec::{Deserialize, Serialize};
 
+/// This trait is used to provide authorization mechanisms for the MlsCentral struct, so that
```

Comment:
Not only. This is just the callback interface. Right now it only contains authorization, but it'll be expanded as needed (i.e. propagating the information that the client was removed from a MLS group through a commit).